contract/src/test/java/io/neow3j/contract/NeoTokenTest.java

```diff
@@ -88,7 +88,6 @@ public void getUnclaimedGas() throws IOException {
 
     @Test
     public void registerCandidate() throws IOException {
-        NeoConfig.setMagicNumber(new byte[]{0x01, 0x03, 0x00, 0x0}); // Magic number 769
```

Comment:
Why was this removed?

Reply:
We looked at this test as an example for setting the magic number and realized that specifically in this test it isn't even necessary.

Reply:
In this test this is not necessary, I looked into it with @csmuller.